beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1690,13 +1658,6 @@ pub static DATA_COLUMN_SIDECAR_GOSSIP_VERIFICATION_TIMES: LazyLock<Result<Histog
             "Full runtime of data column sidecars gossip verification",
         )
     });
-pub static DATA_COLUMNS_SIDECAR_PROCESSING_SUCCESSES: LazyLock<Result<IntCounter>> =
-    LazyLock::new(|| {
-        try_create_int_counter(
-            "beacon_data_column_sidecar_processing_successes_total",
-            "Number of data column sidecars verified for gossip",
-        )
-    });
```

Comment:
Note: another metric with the same name `beacon_data_column_sidecar_processing_successes_total` exists:

https://github.com/sigp/lighthouse/blob/86c65d6dcf533b5fcec928a2ef39704fc840d3ff/beacon_node/beacon_chain/src/metrics.rs#L1647-L1653